packages/dashboard-core-plugins/src/panels/ConsolePanel.jsx

```diff
@@ -272,7 +272,8 @@ export class ConsolePanel extends PureComponent {
       unzip,
     } = this.props;
     const { consoleSettings, error, objectMap } = this.state;
-    const { config, session } = sessionWrapper;
+    // eslint-disable-next-line react/prop-types
```

Comment:
Why is this rule disabled here? Seems you should be providing a prop-type for this

Reply:
So, I'm expecting the dashboard-core-plugins conversion to be merged first, so there wouldn't be a point in adding prop-type when it's just gonna get removed when I merge it